README.md

```diff
@@ -137,36 +138,86 @@ See [Common Parameters](#commonargs) for a list of additional parameters, includ
 
-`--upload-modules    -U`  
+`--upload-modules    -U`
 (optional) If specified, uploads Node.js modules from your system to Apigee Edge rather than resolving the modules directly on Apigee Edge (the default behavior).
 
-`--virtualhosts  -v`  
+`--virtualhosts  -v`
+(optional) A comma-separated list of virtual hosts that the deployed app will use. The two most common options are `default` and `secure`. The `default` option is always HTTP and `secure` is always HTTPS. By default, `apigeetool deploynodeapp` uses `default,secure`.
+
+## <a name="deployhostedfunction"></a>deployhostedfunction
+
+Deploys a Hosted Function to Apigee Edge as an API proxy. With your Hosted Function deployed to Edge, you can take advantage of Edge features like security, quotas, caching, analytics, trace tool, and more.
+
+#### Examples
+
+Deploys a Node.js app as a Hosted Function to Apigee Edge.
+
+    apigeetool deployhostedfunction -u sdoe@apigee.com -o sdoe -e test -n 'test-node-app-2' -b /node2
+
+Deploys a Node.js app as a Hosted Function to both the default (HTTP) and secure (HTTPS) virtual hosts.
+
+    apigeetool deployhostedfunction -u sdoe@apigee.com -o sdoe -e test -n 'test-node-app-2' -b /node2 -v default,secure
+
+#### Required parameters
+
+The following parameters are required. However, if any are left unspecified
+on the command line, and if apigeetool is running in an interactive shell,
+then apigeetool will prompt for them.
+
+For example, if you do not specify a password using "-p", apigeetool will
+prompt for the password on the command line.
+
+See [Common Parameters](#commonargs) for a list of additional parameters, including
+the "-u" and "-p" parameters for username and password, and the "-o" parameter
+for organization name, all of which are required.
+
+`--api   -n`
+The name of the API proxy. The name of the API proxy must be unique within an organization. The characters you are allowed to use in the name are restricted to the following: `a-z0-9._\-$ %`.
+
+`--environments  -e`
+(required) The name(s) of the environment(s) to deploy to (comma-delimited).
+
+#### Optional parameters
+
+`--base-path -b`
+(optional) The base path of the API proxy. For example, for this API proxy, the base path is `/example-proxy`: `http://myorg-test.apigee.net/example-proxy/resource1`.
+
+`--directory -d`
+(optional) The path to the root directory of the API proxy on your local system. Will attempt to use current directory is none is specified.
+
+`--import-only   -i`
+(optional) Imports the API proxy to Apigee Edge but does not deploy it.
+
+`--preserve-policies -P`
+(optional) If specified, the highest revision of the existing proxy will be downloaded and the node code in your directory will be overlayed upon it to create a resulting proxy that contains both any existing policies and the node code in the directory. If there is no existing revision, this option will have no effect.
```

Comment:
the `--preserve-policies` option description only mentions overlaying "node code". While thats true right now, in the future it won't be. Should say something like "sources" or "source code"